src/features/Banner/component.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable max-len */
+import React, { Component } from 'react';
+// import components
+import Navbar from '../../shared/Navbar';
+import SearchMovie from '../../shared/SearchMovie';
+import './style.scss';
```

Comment:
let's use css modules instead
https://github.com/css-modules/css-modules

Reply:
Ok this was added. Are we implementing this on every component? I would only expect to have css collisions on some button elements for this application